modules/tinymce/src/plugins/link/test/ts/browser/NormalizeLinkTest.ts

```diff
@@ -0,0 +1,120 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { Optional } from '@ephox/katamari';
+import { KAssert } from '@ephox/katamari-assertions';
+import { TinyHooks } from '@ephox/mcagar';
+import { Hierarchy, SugarElement } from '@ephox/sugar';
+import Editor from 'tinymce/core/api/Editor';
+import { getSelectedLink } from 'tinymce/plugins/link/core/NormalizeLink';
+import Theme from 'tinymce/themes/silver/Theme';
+interface BaseRange {
```

Comment:
Maybe fluff this up a bit add new lines between each block.

Reply:
updated